spec/02-integration/09-hybrid_mode/01-sync_spec.lua

```diff
@@ -17,6 +17,7 @@ for _, strategy in helpers.each_strategy() do
         cluster_cert_key = "spec/fixtures/kong_clustering.key",
         lua_ssl_trusted_certificate = "spec/fixtures/kong_clustering.crt",
         database = strategy,
+        db_update_frequency = 1,
```

Comment:
For testing purpose, this can be set lower, say to `0.1` second to help test converges quicker.